content/community/kug-2023.md

```diff
@@ -0,0 +1,128 @@
+
+## Venue
+
+The Kokkos User Group Meeting will be held at the UNM Continuing Education Center at 1634 University Blvd NE, Albuquerque, NM 87131.
+* Free parking is available.
+* A light lunch and snacks will be provided. 
+
+## Program
+
+The Kokkos User Group Meeting will consist of a mix of Kokkos Team provided talks and user contributions.
+
+#### Day 1 - Tuesday 12th of December
+
+* 9am - 5pm
+* Kokkos Community Development
+  * Post ECP future of the Kokkos Community
+  * Open Governance and Project Sustainment
+  * Developing Partnerships
+* Kokkos Primary Project Updates
+  * Kokkos Core
+  * Kokkos Kernels
+  * Kokkos Tools
+
+#### Day 2 - Wednesday 13th of December
+
+* 9am - 5pm
+* Kokkos Secondary Project Updates
+  * PyKokkos
+  * Resilience
+  * Remote Spaces
+  * Fortran Interop
+* Community Talks
+
+#### Day 3 - Thursday 14th of December
+
+* 9am - 5pm
+* Community Talks  
+
+#### Day 4 - Friday 15th of December
+
+* 9am - 1pm
+* Feedback session
+  * What works and doesn't work?
+  * Most common stumbling blocks.
+  * What features would you like to see?
+  * 5min contributed lightning talks + discussion
+
+## Contributing
+
+The Kokkos Team is looking for contributions from users giving insight into their experience.
+
+We are looking for contributions in various areas:
+
+* Use-case presentations
+  * Domain specific algorithms
+  * Domain specific data structures
+  * Clever/Fun ways to use Kokkos
+  * Best practices
+* Performance Analysis
+  * Comparing different hardware
+  * Comparison to native implementations
+  * Performance optimization tricks
+  * Issues where Kokkos prevented good performance
+* Development practices and debugging
+  * Software engineering approaches for Kokkos
+  * Build system integration experience
+  * Best practices for debugging Kokkos applications
+* Experiences with teaching/learning Kokkos
+  * Beginners experience: how was the onboarding, what could have helped?
+  * Teachers experience: what additional material would be useful, how does the existing one hold up?
+* Student contributions
+  * Talks by undergraduate and graduate students, who wish to present their work with Kokkos
+
+Format: 20 min talks and potentially 5 min lightning talks
+
+**Please indicate your interest in contributing a talk by providing a talk title in the comment section of the registration form.**
+
+## Registration
+
+You must register to attend. Attendance is capped with a first-come first-served policy.
+
+A registration fee of $125 will be charged. 
+
+*Registration Link:* 
+
+## Accomodation
+
+Albuquerque is NOT an inherently walkable city, however there are two decent hotels close enough that a car wouldn't be strictly necessary.
+Besides those there are many good hotels throughout Albuquerque. Any place *east* of the River (i.e. in Albuquerque proper) will be fine and at most 15-20mins by car.
+There is *NO* usable public transport other than ride services.
+
```

Comment:
Instead of 112 - 113, how about:

- For our European guests, Albuquerque is a typical grid-based western US city best navigated and explored in a car.  There are, however, pedestrian - friendly districts (below).   

- Pedestrian-friendly districts in Albuquerque:
  - [Old Town Spanish Settlement Dating from the Early 1700](https://www.albuquerqueoldtown.com/)
  - [University of New Mexico (UNM) vicinity](https://www.tripadvisor.com/RestaurantsNear-g60933-d144326-University_of_New_Mexico-Albuquerque_New_Mexico.html)
  - [Nob Hill, East of UNM](https://nobhillmainstreet.org/)
  - [ABQ Uptown](https://www.simon.com/mall/abq-uptown)


- Hotels in the vicinity of the conference venue:
  - [Hotel Zazz in Nob Hill](https://www.hotelzazz.com/about)
  - [Crown Plaza UNM](https://crowneplaza.com)
  - [Home2 Albuquerque Downtown](https://www.hilton.com/en/hotels/abqduht-home2-suites-albuquerque-downtown-university/)
  - [Hyatt Place ABQ Uptown](https://hyatt.com)

- Transportation:
  - We *do not* recommend public transportation in Albuquerque 😏 
  - App-based ride services, such as Uber and Lyft are reliable
  - Many car rental options at the Albuquerque Sunport